schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+CREATE TABLE animals(
+    id int primary key not null,
```

Comment:
## [OPTIONAL]

- [ ] Great Job :clap: on making the id of the animal table to not be null as well as to be a primary key, however, I would suggest you also make the Id to autoincremented. check this example :point_down: 
```sql
id int NOT NULL IDENTITY(1,1)
```